docs/planning/timeline-controls.mdx

```diff
@@ -0,0 +1,34 @@
+import timelineIconTray from './assets/timeline-icon-tray.png';
+
+# Timeline Controls
+
+Here we describe the timeline rendering and interaction options that are not part of the Aerie UI View Above the timeline is a tray of icon buttons that can be used to control how data is rendered and interacted with on the timeline.
+
+<figure>
+  <img alt="Aerie UI - Timeline Icon Tray" src={timelineIconTray} />
+</figure>
+
+The following controls are available:
```

Comment:
Minor: Proposal: edit the image to be labelled with numbers, and then have this list be in numerical order. It will help users associate the icons with the entries.